reliability/node_tuning_operator.py

```diff
@@ -0,0 +1,344 @@
+#!/usr/bin/env python
+
+import subprocess
+import time
+
+#############################################################
+# Test: Node Tuning Operator: core functionality is working #
+#############################################################
+
+# TODO - turn off colors on demand
+# Setting the output colors
+
+
+red = "\33[31m"
+on_red = "\33[41m"
+blue = "\33[34m"
+on_blue = "\33[44m"
+green = "\33[32"
+on_green = "\33[42m"
+reset = "\33[0m"
+
+
+# TODO - move print/execute steps to some new utils file of find what is in SVT already
+def print_title(title):
+    print("Running test: {}{}{}".format(on_blue, title, reset))
+
+
+def print_step(step_description):
+    print("\nStep: {}{}{}".format(on_blue, step_description, reset))
+
+
+def print_command(command_description):
+    print("Executing command: {}{}{}".format(blue, command_description, reset))
+
+
+def print_warning(warning):
+    print("{}{}{}".format(red, warning, reset))
+
+
+def passed(description):
+    print("{}STEP PASSED{}".format(on_green, reset))
+    if description is not None:
+        print("{}{}{}".format(green, description, reset))
+
+
+def fail(description):
+    print("{}STEP FAILED{}".format(on_red, reset))
+    if description is not None:
+        print("{}{}{}".format(red, description, reset))
+    raise SystemExit
+
+
+def execute_command(command_to_execute):
+    print_command(command_to_execute)
+    try:
+        value_to_return = subprocess.check_output(command_to_execute, shell=True)
+    except subprocess.CalledProcessError as exc:
+        value_to_return = exc.output
+    return value_to_return
+
+
+def execute_command_on_node(externall_address, node_address, command_to_execute):
+    command_on_node = "ssh -i ~/.ssh/libra.pem -o ProxyCommand='ssh -A -i ~/.ssh/libra.pem -W %h:%p core@{}' core@{} {}".format(externall_address, node_address, command_to_execute)
```

Comment:
When did the libra key get copied to the proxy node?
I would use a var for the path of the key because other people may use a different key.

Reply:
Path to key is no longer hardcoded.
It's not copied - it's using the same.